server/planning/agendas_async/module.py

```diff
@@ -10,5 +10,9 @@
     name="agenda",
     data_class=AgendasResourceModel,
     service=AgendasAsyncService,
-    rest_endpoints=RestEndpointConfig(resource_methods=["GET", "POST"], item_methods=["GET", "PATCH", "DELETE"]),
+    rest_endpoints=RestEndpointConfig(
+        resource_methods=["GET", "POST"],
+        item_methods=["GET", "PATCH", "DELETE"],
+        enable_cors=True,
```

Comment:
We can leave these `enable_cores=True` in here, though in my PR I decided it's best we enable by default and disable where required (as more resources would use them then not)